app/models/hyrax/pcdm_collection.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Hyrax
+  ##
+  # Valkyrie model for Collection domain objects in the Hydra Works model.
+  class PcdmCollection < Hyrax::Resource
+    include Hyrax::Schema(:core_metadata)
+
+    attribute :collection_type_gid, Valkyrie::Types::String
+    attribute :member_ids, Valkyrie::Types::Array.of(Valkyrie::Types::ID).meta(ordered: true)
```

Comment:
Is that fine to add `collection_type`, `collection_type=` in the model?

Reply:
We don't supply a `collection_type` setter without the `_gid` suffix in the current `CollectionBehavior` implementation. There *is* a ticket to refine that implementation, though it might be a `wontfix` until after the ActiveFedora behavior is removed.

I'm not necessarily against putting them in now, but I'm a bit worried about the impact on `Wings`. Keeping symmetry between Valkyrie and ActiveFedora properties simplifies things a bit.

Reply:
Gotcha. Sounds reasonable.